src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -248,8 +248,11 @@ def _set_batches(self):
 
         for i, expt in enumerate(self._experiments):
             expt.scan.set_batch_offset(i * 10 ** n)
-            # This may be a different scan instance ¯\_(ツ)_/¯
-            expt.imageset.get_scan().set_batch_offset(expt.scan.get_batch_offset())
+            try:
+                # This may be a different scan instance ¯\_(ツ)_/¯
+                expt.imageset.get_scan().set_batch_offset(expt.scan.get_batch_offset())
+            except AttributeError:
+                pass
```

Comment:
Is this just catching the case where `expt.imageset is None`? If so, could this just be a simple `if` statement rather than `try`/`except`? 
```suggestion
            if expt.imageset:
                # This may be a different scan instance ¯\_(ツ)_/¯
                expt.imageset.get_scan().set_batch_offset(expt.scan.get_batch_offset())
```